### ISSUE TITLE
There are no commands defined in the "plesk-ext-laravel" namespace.

### ISSUE THREAD

**Issue by Spharian** [plesk/ext-laravel-integration#4]
Hello,

I'm suddenly getting the following error whenever I browse my Laravel application on Plesk:

`There are no commands defined in the "plesk-ext-laravel" namespace.
`

Is it due to an update or something else?

Kind regards

**Comment by martinallen-exe** [comment]
We're also seeing this repeatedly on our Laravel Application hosted in Plesk. 

```
[2023-01-27 15:09:44] development.ERROR: There are no commands defined in the "plesk-ext-laravel" namespace. {"exception":"[object]
(Symfony\\Component\\Console\\Exception\\NamespaceNotFoundException(code:
0): There are no commands defined in the \"plesk-ext-laravel\"
namespace. at
/var/www/vhosts/xxxx/vendor/symfony/console/Application.php:624)
[stacktrace]
#0
/var/www/vhosts/xxxx/vendor/symfony/console/Application.php(675):
Symfony\\Component\\Console\\Application->findNamespace()
#1
/var/www/vhosts/xxxx/vendor/symfony/console/Application.php(259):
Symfony\\Component\\Console\\Application->find()
#2
/var/www/vhosts/xxxx/vendor/symfony/console/Application.php(171):
Symfony\\Component\\Console\\Application->doRun()
#3
/var/www/vhosts/xxx/vendor/laravel/framework/src/Illuminate/Console/Application.php(111):
Symfony\\Component\\Console\\Application->run()
#4
/var/www/vhostsxxxx/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(155):
Illuminate\\Console\\Application->run()
#5 /var/www/vhosts/xxxx/artisan(37):
Illuminate\\Foundation\\Console\\Kernel->handle()
#6 {main}
"} 
```

Looks like its occurring when `artisan schedule:run` is being called every minute??

**Comment by Spharian** [comment]
I'm glad someone responds to my issue @martinallen-exe wasn't sure it was being checked haha.
For me, the error comes whenever I browse the plesk project after clicking "Manage Laravel Application".

**Comment by martinallen-exe** [comment]
@Spharian your correct, just checked and it does occur whenever we navigate to the 'Manage Laravel Application' of our domain in Plesk. Im guessing we are only seeing the issue because we have automated error logging notifying us immediately. 
We have the Queue setting turned off, not sure if that makes a difference.

**Comment by Spharian** [comment]
@martinallen-exe yeah I'm notified on Slack for the errors, that's why it's annoying. Seems related to the recent Queue update, which is turned off on my side too.

**Comment by Spharian** [comment]
@martinallen-exe FYI, doing `composer require plesk/ext-laravel-integration` and deploying it on the server solves the error. The package should be optional but looks like Plesk doesn't care about its customers and open sourced code 👎

**Comment by ferrumfist** [comment]
This problem is caused by the latest update of the Laravel Toolkit. The problem has already been fixed in version 1.4.3 which will be released soon
